src/torchmetrics/classification/roc.py

```diff
@@ -105,6 +115,52 @@ def compute(self) -> Tuple[Tensor, Tensor, Tensor]:
         state = [dim_zero_cat(self.preds), dim_zero_cat(self.target)] if self.thresholds is None else self.confmat
         return _binary_roc_compute(state, self.thresholds)
 
+    def compute_auroc(self) -> Tensor:
```

Comment:
should we make this a private method?
also please add a small docstring

Reply:
Sounds fine with me making this a private method. I will add a commit over the weekend.